File: Algoritmo(C)/Alg_S6_6.c
#include <stdio.h>
int main(){
    //variables
    int horas, salario = 0, exc = 0;
    //input
    printf("Digite a quantidade de horas trabalhadas: ");
    scanf("%d",&horas);
    //process
    if (horas > 50){
        exc = ((horas - 50) * 20);
        salario = (500 + exc);
    }else{
        salario = (horas * 10);
    }
    //output
    printf("O salario total e de R$%d e o excesso foi de R$%d",salario,exc);
}

File: Algoritmo(Python)/Alg_S6_6.py
#variables
exc = 0
#input
cod = int(input("Digite o código do funcionário: "))
horas = int(input("Digite a quantidade de horas trabalhadas: "))
#process
if(horas > 50):
    exc = ((horas - 50) * 20)
    salario = (500 + exc)
else:
    salario = (horas * 10)
print("Salário TOTAL: {0}".format(salario))
print("Salário EXCEDENTE: {0}".format(exc))
